data-azure-cosmos/src/test/java/io/micronaut/data/azure/entities/Family.java

```diff
@@ -0,0 +1,57 @@
+package io.micronaut.data.azure.entities;
+
+import io.micronaut.data.annotation.Id;
+import io.micronaut.data.annotation.MappedEntity;
+import io.micronaut.data.annotation.Relation;
+import io.micronaut.data.cosmos.annotation.Container;
+import io.micronaut.data.cosmos.annotation.PartitionKey;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@Container
+@MappedEntity
+public class Family {
```

Comment:
Please remove the need to have all the mapped entities for Cosmos annotated as `@Container`.
The correct way is the Micronaut Data Cosmos configuration will define a set of packages, and those are picked up and treated as Mapped Entity <-> Cosmos Container where the name of the entity is the container name.

Reply:
Removed and added ability to configure container properties through config.